src/main/java/cinema/model/dto/user/UserRequestDto.java

```diff
@@ -1,8 +1,32 @@
 package cinema.model.dto.user;
 
+import cinema.validation.ValidateEmail;
+import cinema.validation.ValidatePassword;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@ValidatePassword.List({
+        @ValidatePassword(
+                field = "password",
+                fieldMatch = "repeatPassword",
+                message = "Passwords don't match."
+        )
+})
 public class UserRequestDto {
+    @ValidateEmail
     private String email;
+    @Size(max = 50)
+    @NotNull
```

Comment:
You accept null value in your validator and telling it can't be null here.
But, in any case, you have constraint annotation for password, no need to use standard annotation for fields, keep all your constraints in `Validator.class`